.editorconfig

```diff
@@ -0,0 +1,5 @@
+[*]
+charset = utf-8
+indent_style = tab
+indent_size = 5
```

Comment:
Why did you choose these values? I see the existing project identation is different from these values (indent_style is **space** and indent_size is **2**)